integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/receipt/MessageStatusRepository.java

```diff
@@ -1,15 +1,66 @@
 package no.difi.meldingsutveksling.receipt;
 
-import org.springframework.data.querydsl.QueryDslPredicateExecutor;
-import org.springframework.data.repository.CrudRepository;
+import com.querydsl.core.BooleanBuilder;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.EntityGraph;
+import org.springframework.data.querydsl.QuerydslPredicateExecutor;
+import org.springframework.data.querydsl.binding.QuerydslBinderCustomizer;
+import org.springframework.data.querydsl.binding.QuerydslBindings;
+import org.springframework.data.repository.PagingAndSortingRepository;
 
-import java.util.List;
 import java.util.Optional;
 
-public interface MessageStatusRepository extends CrudRepository<MessageStatus, String>, QueryDslPredicateExecutor<MessageStatus> {
-    Optional<MessageStatus> findByStatId(Integer statId);
-    List<MessageStatus> findAllByConvId(Integer convId);
-    List<MessageStatus> findByStatIdGreaterThanEqual(Integer statId);
-    List<MessageStatus> findAllByConvIdAndStatIdGreaterThanEqual(Integer convId, Integer recId);
+public interface MessageStatusRepository extends PagingAndSortingRepository<MessageStatus, Long>,
+        QuerydslPredicateExecutor<MessageStatus>,
+        QuerydslBinderCustomizer<QMessageStatus> {
+
+    @EntityGraph("MessageStatus.conversation")
+    Optional<MessageStatus> findById(Long id);
+
+//    @EntityGraph("MessageStatus.conversation")
+//    List<MessageStatus> findAllByConversation_Id(Long convId);
+//
+//    @EntityGraph("MessageStatus.conversation")
+//    List<MessageStatus> findByIdGreaterThanEqual(Long id);
+//
+//    @EntityGraph("MessageStatus.conversation")
+//    List<MessageStatus> findAllByConversation_IdAndIdGreaterThanEqual(Long convId, Long id);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)